test/beforeRequest.test.js

```diff
@@ -125,22 +125,22 @@ function mockXHRCommitments() {
     mockXHR(this);
     this.status = 200;
     this.readyState = 4;
-    this.responseText = `{"CF":{"dev":{"G": "` + testDevG + `","H": "` + testDevH + `"},"1.0":{"G":"` + testG + `","H":"` + testH + `"},"1.1":{"G":"new_11_commitment_g","H":"new_11_commitment_h"}}}`;
+    this.responseText = "{\"CF\":{\"dev\":{\"G\": \"" + testDevG + "\",\"H\": \"" + testDevH + "\"},\"1.0\":{\"G\":\"" + testG + "\",\"H\":\"" + testH + "\"},\"1.1\":{\"G\":\"new_11_commitment_g\",\"H\":\"new_11_commitment_h\"}}}";
```

Comment:
I guess that the linter was complaining about concatenation of template literals. Have you tried something like:
```js
this.responseText = `{"CF":{"dev":{"G":"${testDevG}", ...`
```

I think that would be more readable than escaping quotes. Alternatively, you could use `JSON.stringify` on the object literal.

The same comment applies to storedTokens below (and possibly other places).